contracts/math/SafeMath.sol

```diff
@@ -45,7 +75,17 @@ library SafeMath {
     }
 
     /**
-    * @dev Adds two numbers, reverts on overflow.
+    * @dev Subtracts two signed integers, reverts on overflow (i.e. if subtrahend is greater than minuend).
```

Comment:
This description is not accurate for signed subtraction, e.g. `2 - 3` works fine.